contracts/modules/interfaces/ISuperMinterV1_1.sol

```diff
@@ -137,21 +143,34 @@ interface ISuperMinterV1_1 is IERC165 {
         uint256 requiredEtherValue;
         // The price per token.
         uint256 unitPrice;
-        // The total platform fees.
-        uint256 platformFee;
-        // The total platform flat fees.
-        uint256 platformFlatFee;
-        // The total affiliate fees.
-        uint256 affiliateFee;
+        // The final artist fee.
+        uint256 finalArtistFee;
+        // The final platform fee.
+        uint256 finalPlatformFee;
+        // The total affiliate fee.
+        uint256 finalAffiliateFee;
+        // The final cheap mint fee.
+        uint256 finalCheapMintFee;
```

Comment:
curious why this is included in the minted struct when other things like affiliateIncentive is not?

Reply:
Good call out. I added `finalAffiliateIncentive` to the minted log and renamed `finalCheapMintFee` -> `finalCheapMintIncentive` in the minted log.